tools/ramclustr/test-data/test-spectra_csv.msp

```diff
@@ -0,0 +1,357 @@
+NAME:C01
```

Comment:
Why is this file here?

Reply:
@martenson, thanks for the note. Forgot to write that output to the wrapper, fixed that in the last commit.